src/Article.jsx

```diff
@@ -0,0 +1,28 @@
+import { useEffect, useState } from "react";
+import { getArticleById } from "./utils";
+import CommentSection from "./CommentsSection";
+import { Box } from "@mui/material";
+import FullArticleCard from "./FullArticleCard";
+
+export default function Article() {
+  const [article, setArticle] = useState([]);
+  const articleId = window.location.href.split("/").pop();
```

Comment:
it would be easier to access the id using useParams(), see here: https://reactrouter.com/en/main/hooks/use-params